rsrc.go

```diff
@@ -69,17 +47,17 @@ func main() {
 	var err error
 	switch {
 	case fnamein != "" || fnameico != "":
-		err = run(fnamein, fnameico, fnameout, arch)
+		err = rsrc.Embed(fnameout, arch, fnamein, fnameico)
 	case fnamedata != "":
-		err = rundata(fnamedata, fnameout, arch)
+		err = embeddata(fnameout, arch, fnamedata)
 	}
 	if err != nil {
 		fmt.Fprintln(os.Stderr, err)
 		os.Exit(1)
 	}
 }
 
-func rundata(fnamedata, fnameout, arch string) error {
+func embeddata(fnameout, arch, fnamedata string) error {
```

Comment:
Go style suggests using mixed caps, so this should be `embedData` rather than `embeddata`.

Reference: https://github.com/golang/go/wiki/CodeReviewComments#mixed-caps.

Reply:
Curiously, IIRC the stdlib seems to have quite a lot of all-lowercase identifiers, esp. in variable names; not sure what's the policy on helper methods now, but I seem to recall in the old times they too were all-lowercase quite often. Or do I misremember? That's why I'm personally somewhat on the fence as to this, and that's the reason why this codebase looks this way originally.

That said, I understand the general sentiment. In this particular situation, I'm ok whichever way @asticode prefers, though in fact with slight preference to the proposed mixed caps. Just please don't start renaming variables in this commit.